### DISCUSSION TITLE
How to add custom tags?

### DISCUSSION THREAD

**Discussion by syntaxterror9** [zanllp/infinite-image-browsing#465] [Q&A]
First of all, thank you for this wonderful extension. It quickly became one of my most-used SD extensions.

I was trying to find out how to add custom tags. For example, I want to flag images that need inpainting, img2img, or seed refinement. Is it possible to add them? Can I also assign keyboard shortcuts to them?

Thanks again!

**Comment by zanllp** [comment]
Click here to add a custom tag
<img width="817" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/3928d1f7-cd6b-4bda-a972-c0b8d0847f74">

You can assign keyboard shortcuts to them here.

<img width="1352" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/3b6799ac-37b7-4a66-94a2-4623b967c619">

**Reply by molegris** [reply]
OMG!  Thank you.
I've been searching for this for months.  Never thought the "Add custom tag" feature would be in the Search tool.
Maybe you should add a small note/reminder in the Global settings tab.
In anyway, it's a great extension!  love It!

**Comment by syntaxterror9** [comment]
Great! Thank you 😊
